libraries/Microsoft.Bot.Builder/Adapters/TestFlow.cs

```diff
@@ -165,6 +165,36 @@ public TestFlow AssertReply(string expected, string description = null, uint tim
             return AssertReply(_adapter.MakeActivity(expected), description, timeout);
         }
 
+        /// <summary>
+        /// Adds an assertion that the turn processing logic responds as expected.
+        /// </summary>
+        /// <param name="expected">The part of the expected text of a message from the bot.</param>
+        /// <param name="description">A message to send if the actual response is not as expected.</param>
+        /// <param name="timeout">The amount of time in milliseconds within which a response is expected.</param>
+        /// <returns>A new <see cref="TestFlow"/> object that appends this assertion to the modeled exchange.</returns>
+        /// <remarks>This method does not modify the original <see cref="TestFlow"/> object.</remarks>
+        /// <exception cref="Exception">The bot did not respond as expected.</exception>
+        public TestFlow AssertReplyContain(string expected, string description = null, uint timeout = 3000)
```

Comment:
Typo, method name should probably be AssertReplyContain**s**.

Reply:
Resolved.
https://github.com/microsoft/botbuilder-dotnet/pull/1922/commits/f8a95c0a462b0359af59290fc70fd1548eae3efd